### PR TITLE
Adding error metadata to DialogOpenResponse

### PR DESCRIPTION
**Pull request by louis-pre**
Hello! 
While trying to work with the dialog.open method, I encountered formatting issues. Slack reports them in the response they return but it was not accessible through the DialogOpenResponse object.
(see https://api.slack.com/methods/dialog.open, response_metadata)

I have added them to the dialog response and named the object ErrorMetadata because ResponseMetadata already exists (for chat responses, as I understand it).

I hope it does not break any of your conventions :)

### CODE REVIEW

File: src/main/java/com/github/seratch/jslack/api/model/ErrorMetadata.java [hunk 1]
```diff
@@ -0,0 +1,12 @@
+package com.github.seratch.jslack.api.model;
+
+import lombok.Data;
+
+import java.util.List;
+
+@Data
+public class ErrorMetadata {
```
Comment (seratch) [inline src/main/java/com/github/seratch/jslack/api/model/ErrorMetadata.java]: How about renaming this to `com.github.seratch.jslack.api.model.dialog.DialogResponseMetadata`?

**Comment by louis-pre** [comment]
Done! 
Can you do a Maven deployment when merge?

Thanks a lot :)

File: src/main/java/com/github/seratch/jslack/api/model/ErrorMetadata.java (hunk 1 above)
Comment (seratch) [reply src/main/java/com/github/seratch/jslack/api/model/ErrorMetadata.java]: Please change its package as well.

Comment (seratch) [reply src/main/java/com/github/seratch/jslack/api/model/ErrorMetadata.java]: Please change its package as well.

**Comment by seratch** [comment]
I wil do. After you change the package to place the class, I will merge this PR.

**Comment by louis-pre** [comment]
Thanks, sorry about that.

**Comment by seratch** [comment]
Thank you for your contribution. I've released 1.0.23 which contains your contribution :tada:

**Pull request merged**